src/openrct2/platform/Shared.cpp

```diff
@@ -198,6 +199,24 @@ uint8_t platform_get_currency_value(const char* currCode)
     return CURRENCY_POUNDS;
 }
 
+#ifndef _WIN32
+std::string platform_sanitise_filename(const std::string& path)
+{
+    auto sanitised = path;
+
+    std::vector<std::string::value_type> prohibited = { '/' };
```

Comment:
Feels off to have a single element vector with dynamic memory allocation, too bad theres no std::make_array (yet), could at least make this const.